Packs/ipinfo/Integrations/integration-Ipinfo.yml

```diff
@@ -80,20 +108,73 @@ script:
       required: true
       default: true
       description: IP address to query. E.g. !ip 1.1.1.1
+      isArray: false
     outputs:
     - contextPath: IP.Address
       description: The IP address
+      type: string
     - contextPath: IP.Hostname
       description: The IP hostname
+      type: string
     - contextPath: IP.ASN
       description: The IP ASN
+      type: string
     - contextPath: IP.Geo.Location
       description: The IP geographic location in coordinates
+      type: string
     - contextPath: IP.Geo.Country
       description: The IP country
+      type: string
     - contextPath: IP.Geo.Description
       description: The IP location as <City, Region, Postal Code, Country>
-    description: Check IP reputation (when information is available, returns a JSON with details).  Uses all configured Threat Intelligence feeds
+      type: string
+    - contextPath: IP.ASOwner
+      description: The IP ASN owner
+      type: string
+    - contextPath: IP.FeedRelatedIndicators
+      description: Related Indicators
+      type: string
+    - contextPath: IP.Tags
+      description: the tags
+      type: string
+    - contextPath: IP.Registrar.Abuse.Address
+      description: The Abuse Address
+      type: string
+    - contextPath: IP.Registrar.Abuse.Country
+      description: The Abuse Country
+      type: string
+    - contextPath: IP.Registrar.Abuse.Name
+      description: The Abuse Name
+      type: string
+    - contextPath: IP.Registrar.Abuse.Network
+      description: The Abuse Network
+      type: string
+    - contextPath: IP.Registrar.Abuse.Phone
+      description: The Abuse Phone
+      type: string
+    - contextPath: IP.Registrar.Abuse.Email
+      description: The Abuse Email
+      type: string
+    - contextPath: IP.Organization.Name
+      description: The Organization Name
+      type: string
+    - contextPath: IP.Organization.Type
+      description: The Organization Type
+      type: string
+    - contextPath: DBotScore.Indicator
```

Comment:
DbotScore objects aren't supposed to be directly associated with the indicator type (just talked about it the other day with aviad) - so please remove these.